entity-types/uninstrumented-awslambdaalias/definition.yml

```diff
@@ -0,0 +1,6 @@
+domain: UNINSTRUMENTED
```

Comment:
Uninstrumented entities get created automatically via the relationship definition: https://github.com/newrelic/entity-definitions/pull/1726/files#diff-2fb16cc8dafd02f461df0b26bd5599a1b11efab05a9364ccdcf58771966161ecR13

Please take a look at our docs here: https://github.com/newrelic/entity-definitions/blob/main/docs/relationships/candidate_relationships.md#onmiss

Reply:
Please remove those two uninstrumented definitions

Reply:
Understood. Removed the files